src/components/RevokeContainer/RevokedScreen/RevokedScreen.tsx

```diff
@@ -0,0 +1,192 @@
+import { CheckCircle, XCircle, Download } from "react-feather";
+import { Button, LoaderSpinner } from "@govtechsg/tradetrust-ui-components";
+import { Title } from "../../UI/Title";
+import { generateZipFile, generateFileName, getFileSize } from "../../../utils";
+import { RevokeTag } from "../RevokeTag/RevokeTag";
+
+interface RevokeScreenProps {
+  config: Config;
+  revokeDocuments: any[];
+  revokeAnotherDocument: () => void;
+  onLogout: () => void;
+  fileName: string;
+}
+
+export const RevokedScreen: FunctionComponent<RevokeScreenProps> = ({
+  config,
+  revokeDocuments,
+  revokeAnotherDocument,
+  onLogout,
+  fileName,
+}) => {
+  const { revoke, revokeState, revokedDocuments, failedRevokedDocuments, pendingRevokeDocuments } = useRevokeQueue(
+    config,
+    revokeDocuments
+  );
+
+  useEffect(() => {
+    revoke();
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const getDisplayTitle = (): ReactElement => {
+    switch (revokeState) {
+      case RevokeState.PENDING:
+        return (
+          <>
+            <LoaderSpinner className="mr-2" width="24px" primary="#00cbbc" secondary="#e2e8f0" />
+            Revoking document(s)...
+          </>
+        );
+
+      case RevokeState.CONFIRMED:
+        if (revokedDocuments.length > 0) {
+          return (
+            <>
+              <CheckCircle className="mr-2 text-teal" />
+              Document(s) revoked successfully
+            </>
+          );
+        } else {
+          return (
+            <>
+              <XCircle className="mr-2 text-red" />
+              Document(s) failed to revoke
+            </>
+          );
+        }
+
+      default:
+        return <>Please wait while we prepare your document(s)</>;
+    }
+  };
+
+  const allFailedRevokedDocuments = failedRevokedDocuments.map((failedJob) => failedJob.documents).flat();
+
+  const formattedErrorLog = failedRevokedDocuments.map((failedJob) => {
+    const fileNames = failedJob.documents.map((document) => document.fileName).join(", ");
+    return {
+      files: fileNames,
+      error: failedJob.error,
+    };
+  });
+
+  return (
+    <Wrapper>
+      <ProgressBar step={2} totalSteps={2} title="Revoked Document" />
+      <div className="flex justify-between items-end" data-testid="revoke-screen">
+        <Title className="flex items-center mb-8" data-testid="title">
+          {getDisplayTitle()}
+        </Title>
+        {revokeState === RevokeState.CONFIRMED && (
+          <div>
+            <Button
+              className="bg-white text-orange hover:bg-grey-100 mb-6 mr-4"
+              data-testid="revoke-another-button"
+              onClick={revokeAnotherDocument}
+            >
+              Revoke another document
+            </Button>
+            <Button
+              className="bg-orange text-white hover:bg-orange-600 mb-6"
+              data-testid="logout-button"
+              onClick={onLogout}
+            >
+              Logout
+            </Button>
```

Comment:
why do we have another logout button when there is already 1 in navbar? double checked, can remove

Reply:
ill remove this one here and the published one will be in another PR